scripts/irods/test/test_all_rules.py

```diff
@@ -1502,34 +1502,54 @@ def test_msiRemoveUserFromGroup__issue_7165(self):
         # fed into the rules. This is necessary for testing unqualified usernames and qualified
         # usernames.
         rule_map = {
-            'irods_rule_engine_plugin-irods_rule_language': textwrap.dedent(f'''
-                msiRemoveUserFromGroup('{group}', '{self.user0.username}', '{{}}')
+            'irods_rule_engine_plugin-irods_rule_language': textwrap.dedent('''
+                issue_7165 {{
+                    msiRemoveUserFromGroup('{0}', '{1}', '{2}')
+                }}
+
+                INPUT null
+                OUTPUT ruleExecOut
```

Comment:
Is this different than using the cmdline params (i.e. `irule ... null ruleExecOut`)?

Reply:
For native rules, this is no different. But you cannot execute python rules the same way (as far as I know), so it has to go into a file or get configured in the rulebase and called from a native rule. So, we can either split up this test based on REP type, use different `irule` invocations based on REP type, or just put the rule in a file and execute it the same way for both REP types (this is what I implemented). Open to other options as well.

Reply:
I was just curious. What you've implemented is good.

Reply:
Okey dokey, sounds good